allensdk/brain_observatory/behavior/data_objects/metadata/ophys_experiment_metadata/ophys_experiment_metadata.py

```diff
@@ -142,7 +145,7 @@ def ophys_session_id(self) -> int:
         return self._ophys_session_id.value
 
     @property
-    def project_code(self) -> Optional[str]:
+    def project_code(self) -> Optional[int]:
```

Comment:
I thought it's a string. Currently it's never `None` (optional) since you fill it with a dummy value.

Reply:
Fixed